.github/workflows/bot.yml

```diff
@@ -3,7 +3,7 @@ on:
     types: [created]
 jobs:
   bot:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.04
```

Comment:
I don't know about this. `ubuntu-latest` is always expected to be just a runner capable of running the actions we have, and the actual environment is defined by the actions themselves.

Not a big deal to update every couple years, but still a manual thing to do.

Reply:
Yup, makes sense, we have a strict policy on dependency pinning (also on runners) but we can revert to `-latest` if that comes more handy